test/pages/products/__snapshots__/index.spec.tsx.snap

```diff
@@ -14,9 +14,139 @@ exports[`Product List renders correctly 1`] = `
       <p
         class="styled__StyledText-tqnj75-5 hCSmtq"
       >
-        5 Products
+        undefined Products
```

Comment:
This is a little concerning and doesn't match what actually renders when running locally.

Reply:
so the reason this is coming in as undefined is because we haven't expanded the mock data to include `meta`.  If you look at the `Table` pagination prop, there's a `totalItems` that's coming from meta. `totalItems: meta?.pagination?.total`

Reply:
specifically, look at how we've handled `list` and think about how we might handle/ add `meta`: https://github.com/bigcommerce/sample-app-nodejs/blob/main/test/mocks/hooks.ts#L28